corehq/apps/export/utils.py

```diff
@@ -88,6 +88,20 @@ def _convert_index_to_path(index):
         return ['data'] + index.split('.')[1:]
 
 
+def _convert_index_to_path_nodes(index):
```

Comment:
Followed up in this [comment](https://github.com/dimagi/commcare-hq/pull/10847#discussion_r56867542), but basically export items still use lists of strings.